### PR TITLE
refactor: kubevirt_vm for python coding style

### PR DESCRIPTION
**Pull request by tripledes**
This commit adds/changes:
* 80c line length when possible
* split main() into few less complex methods
* added docstrings

The idea is to stick to PEP8 where possible, as Ansible forces to have imports below documentation which goes against Python standards.

I've tested this commit with `tests/vm.yml` and seems to work properly. Will do more testing before requesting review.

First step for #21

### CODE REVIEW

File: library/kubevirt_vm.py [hunk 1]
```diff
@@ -56,20 +56,26 @@
         required: false
     pvc:
         description:
-            - Name of a PersistentVolumeClaim existing in the same namespace to use as a base disk for ithe VM.
+            - "Name of a PersistentVolumeClaim existing in the same namespace
+              to use as a base disk for ithe VM."
```
Comment (alexxa) [inline library/kubevirt_vm.py]: s/ithe/the

File: library/kubevirt_vm.py [hunk 2]
```diff
@@ -98,32 +104,180 @@
...
 VERSION = 'v1alpha1'
-REGISTRYDISKS = ['kubevirt/alpine-registry-disk-demo', 'kubevirt/cirros-registry-disk-demo', 'kubevirt/fedora-cloud-registry-disk-demo']
+REGISTRYDISKS = [
+    'kubevirt/alpine-registry-disk-demo',
+    'kubevirt/cirros-registry-disk-demo',
+    'kubevirt/fedora-cloud-registry-disk-demo']
+
+
+def build_vm_definition(params):
+    '''returns a dict() containing a VM definition based on params.'''
```
Comment (alexxa) [inline library/kubevirt_vm.py]: Imperative mood, present tense in all docstring please, i.e. `return, delete, remove, update, build, validate, etc`. No 's'. Thanks.

**Comment by tripledes** [comment]
@alexxa I've fix the your two comments and tested the module against `tests/vm.yml` and `tests/vm_cloudinit.yml`. The VM comes up as expected and switching `state` to `absent` removes the VM also as expected. I feel it's good to be merged. Do you have any other point you'd like me to fix or modify?

**Comment by alexxa** [comment]
Merging. Thank you @tripledes .

**Pull request merged**